fluent.conf.xray

```diff
@@ -24,6 +24,7 @@
   metric_name_key metric_name
   metric_value_key value
   insecure_ssl "#{ENV['SPLUNK_INSECURE_SSL']}"
+  gzip_compression "#{ENV['SPLUNK_COMPRESS_DATA']}"
```

Comment:
no execution interval and request timeout for an x-ray?

Reply:
Good point. I'll add them